apps/dapp/src/components/Pages/Core/DappPages/Dashboard/Metrics/index.tsx

```diff
@@ -84,41 +48,39 @@ const DashboardMetrics = ({ dashboardType }: DashboardMetricsProps) => {
     </>
   );
 
-  const desktopView = (sourceData: Nullable<ArrangedDashboardMetrics>) => (
+  const desktopView = (sourceData: ArrangedDashboardMetrics) => (
     <>
       <MetricsContainer>
-        {sourceData!.metrics.map((row, idx) => (
-          <MetricsRow key={idx}>
-            {row.map((metric, idx) => (
+        <MetricsRow>
+          {sourceData.metrics.map((row) =>
+            row.map((metric, idx) => (
               <Metric key={idx}>
                 <MetricValue>{metric.value}</MetricValue>
                 <MetricTitle>{metric.title}</MetricTitle>
               </Metric>
-            ))}
-          </MetricsRow>
-        ))}
-      </MetricsContainer>
-      <MetricsContainer>
-        {sourceData!.smallMetrics.map((row, idx) => (
-          <MetricsRow key={idx}>
-            {row.map((metric, idx) => (
+            ))
+          )}
+        </MetricsRow>
+        <MetricsRow>
+          {sourceData.smallMetrics.map((row) =>
```

Comment:
<img width="1210" alt="Screenshot 2023-11-16 at 7 33 42 AM" src="https://github.com/TempleDAO/temple/assets/99344331/41e7d65a-02bf-4c45-9891-73c0ff37d755">

The alignment seems off for this row.